src/Extension/Resources/config/task/SprykAddModuleTask.yaml

```diff
@@ -3,7 +3,7 @@ id: "generate:php:add-module"
 short_description: "Generate a new Spryker module in the current project"
 help: ~
 stage: build
-command: "php %sdk_dir%/vendor/bin/spryk-run AddModule %organization% %coreLevel% %module%"
+command: "php %sdk_dir%/vendor/bin/spryk-run AddModuleDefault %organization% %coreLevel% %module%"
```

Comment:
Task version must be updated in order to update it in the existing initialized sdk

Reply:
Ups I forgot about that. I will update it.

Reply:
@pavelmaksimov25 task version has been updated. ;)